README.md

```diff
@@ -696,7 +696,7 @@ Other Style Guides
     > Why? They are confusing to reason about.
 
     ```javascript
-    var b = 1;
+    let b = 1;
```

Comment:
These `vars` are intentional, and the examples do not need to execute nor pass a linter. Please excise this commit.